base/float.jl

```diff
@@ -713,6 +713,8 @@ end
     eps(::Type{Float16}) = $(bitcast(Float16, 0x1400))
     eps(::Type{Float32}) = $(bitcast(Float32, 0x34000000))
     eps(::Type{Float64}) = $(bitcast(Float64, 0x3cb0000000000000))
+    eps(::Type{Complex{T}}) where {T<:AbstractFloat} = eps(T)
+    eps(x::Complex{T}) where {T<:AbstractFloat} = eps(norm(x))
```

Comment:
Is it the most appropriate to define the eps for  the norm of the number? E.g. why not  actually define it in terms of `eps(min(x.re,x.im)`  that way it gives the minimum next largest value?